src/components/Storefront/Footer.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { Heading, Image, Box, Center, Text, Link, Button, HStack, VStack, Icon, Input } from '@chakra-ui/react';
+import {
+  FaFacebookSquare,
+  FaInstagramSquare,
+  FaYoutube,
+  FaLinkedin,
+  FaTwitter,
+  FaChevronCircleRight,
+} from 'react-icons/fa';
+import smallLogo from '../../assets/building_up_footer_small_icon.png';
+
+const Footer = () => {
+  const [subscribtionEmail, setSubscribtionEmail] = useState('');
+
+  const handleSubscriptionEmailChange = e => {
+    e.preventDefault();
+    setSubscribtionEmail(e.target.value);
+    console.log(subscribtionEmail);
+  };
+
+  const handleSubscriptionEmailSubmit = e => {
+    e.preventDefault();
+    alert(subscribtionEmail);
+  };
+
+  return (
+    <Box color="white">
+      <HStack backgroundColor="black" paddingLeft="41px" paddingTop="54px" paddingBottom="84px" paddingRight="105px">
```

Comment:
```suggestion
      <HStack backgroundColor="black" py="80px" px="105px">
```

I picked `80px` for vertical padding cuz i think it looks ok 🤷